pkg/extensions/tbtc/tbtc.go

```diff
@@ -437,6 +439,17 @@ func (t *tbtc) monitorAndAct(
 			return
 		}
 
+		if !t.acquireMonitoringLock(depositAddress, monitoringName) {
+			logger.Warningf(
+				"[%v] monitoring for deposit [%v] is already running; "+
+					"could not start new monitoring instance",
```

Comment:
Maybe just the first part `[%v] monitoring for deposit [%v] is already running`? We do not even bother to start new monitoring because it's already running so it's not that we *could not* start it. 😉